styles/autocomplete.less

```diff
@@ -215,3 +216,15 @@ autocomplete-suggestion-list {
     background: fadeout(@color, 80%)
   }
 }
+.make-type-icon(@type, @color)
+  when (hsvvalue(@color) - hsvvalue(@overlay-background-color) >= 0)
+  and (hsvvalue(@color) - hsvvalue(@overlay-background-color) < 20),
+  (hsvvalue(@overlay-background-color) - hsvvalue(@color) > 0)
+  and (hsvvalue(@overlay-background-color) - hsvvalue(@color) < 20)
```

Comment:
Not sure if I understand it fully, but could the above be simplified with:

```less
when (hsvvalue(@color) - hsvvalue(@overlay-background-color) > -20)
and (hsvvalue(@color) - hsvvalue(@overlay-background-color) < 20)
```

So it would pass if the difference of the two variables is between `-20` and `20`.